util/deltautil/Earthfile

```diff
@@ -1,6 +1,7 @@
 VERSION 0.8
 
-FROM github.com/earthly/earthly+base
+# Note: the tag below is needed for auto-skip.
+FROM github.com/earthly/earthly:tags/v0.8.5+base
```

Comment:
could we just go back to using `../..+base` instead?

Reply:
I think these full target names were added because `deltautil` & `ast` are separate packages. I'm concerned that using relative paths may break something.

Reply:
how about using `ARG EARTHLY_GIT_BRANCH` instead?

e.g. 

```
ARG EARTHLY_GIT_BRANCH
FROM github.com/earthly/earthly:$EARTHLY_GIT_BRANCH+base
```

Reply:
or `EARTHLY_GIT_HASH`

Reply:
Good idea. Branches don't work with auto-skip, but hashes will.

Reply:
Actually, the hash will break local runs for unpushed commits. :thinking:

Reply:
When running locally wouldn't EARTHLY_GIT_HASH point to the local hash that might not exist in the remote repo yet?

Reply:
If we use the tag/version, we can configure Renovate to upgrade the version together with all the other places in the repo.

Reply:
3-4 poor options :cry: 
* Use relative paths and possibly break other code that depends on this
* Use hashes and break builds locally
* Use version tag and have to update (unexpected results for local runs)
* Copy and paste the 3-4 root Earthfile `+base` commands

I think I prefer the last option.

Reply:
I don't see any discussion about why this was done under #2579 and don't see any references to it under our other repos either.

it was initially added as `FROM github.com/earthly/earthly+base`; if instead it was changed to it from a relative import there would be more git history for us to dig into to understand why.

I think we should try using `../..+base` and then if we discover anything breaks then we come back and revert this, and add a comment explaining why.

Reply:
We decided to use relative paths as the full canonical target names did not seem necessary.